config/helpers.js

```diff
@@ -59,9 +59,9 @@ execp = (cmd, opts) => {
  * @param opts See child_process.exec node docs
  * @returns {Promise<number>}
  */
-installDependencies = (opts) => {
-  return execp('yarn -v') // first try to install deps using yarn
-    .then(exitCode => exitCode === 0 ? execp('yarn install', opts) : execp('npm install', opts));
+const installDependencies = (opts) => {
+  return execp('npm -v') // first try to install deps using yarn
```

Comment:
@AnthonyNahas This never works for me, I don't have yarn so had to change it to get it to work